madrone1-sanfrancisco.md

```diff
@@ -0,0 +1,6 @@
+### San Francisco, CA
+#### Food
+ - Burma Superstar
+ - Lers Ros Thai
+
+#### Things to Do :eyes:
```

Comment:
@madrone1 I've been to SF a few times, but there's a lot I haven't seen! What do the locals like to do?